pymc/tests/test_distributions_moments.py

```diff
@@ -800,6 +801,26 @@ def test_categorical_moment(p, size, expected):
     assert_moment_is_expected(model, expected)
 
 
+@pytest.mark.parametrize(
+    "x_points, pdf_points, size, expected",
+    [
+        (np.array([-1, 1]), np.array([0.4, 0.6]), None, 0.2),
+        (np.array([-4, -1, 3, 9, 19]), np.array([0.1 , 0.15, 0.2 , 0.25, 0.3 ]), None, 8),
+        # (np.array([-22, -4, 0, 8, 13]), np.tile(1 / 5, 5), (5, 3), -np.ones((5, 3))),
```

Comment:
The `result` from the moment test is always 1.54589372 despite their dot product being 8. Any thoughts on why I am getting the same, constant error? This may be related to the error that I am encountering in `StickBreakingWeights`

Reply:
Solved, the answer to this is that `Interpolated` "connects" the $$(x, f(x))$$ through interpolation. As such, we need to divide the pdf values by `Z`.